portal/src/main/scala/cromwell/pipeline/datastorage/dto/User.scala

```diff
@@ -1,16 +1,24 @@
 package cromwell.pipeline.datastorage.dto
 
+import play.api.libs.json.{Json, OFormat}
 import slick.lifted.MappedTo
 
 final case class User(
-  userId: UserId,
-  email: String,
-  passwordHash: String,
-  passwordSalt: String,
-  firstName: String,
-  lastName: String,
-  profilePicture: Option[ProfilePicture] = None
-)
+                       userId: UserId,
+                       email: String,
+                       passwordHash: String,
+                       passwordSalt: String,
+                       firstName: String,
+                       lastName: String,
+                       profilePicture: Option[ProfilePicture] = None
+                     )
 
 final case class UserId(value: String) extends MappedTo[String]
+object UserId {
+  implicit val userIdFormat: OFormat[UserId] = Json.format[UserId]
```

Comment:
Same problem as [here](https://github.com/EpamLifeSciencesTeam/cmwl_pipeline/pull/16/files#r354549743).
We should discuss how `UserId` should be serialized during our daily meeting.

Reply:
done